langchain/src/chains/combine_docs_chain.ts

```diff
@@ -326,6 +327,9 @@ export class RefineDocumentsChain
   initialResponseName = "existing_answer";
 
   refineLLMChain: LLMChain;
+  
+  progressEmitter?: (progress: number) => Promise<void>
```

Comment:
We've called something similar `onDocumentLoaded` here:

https://github.com/langchain-ai/langchainjs/blob/06a39117aae61fcbf26a537de4bad8a156bbb9ab/langchain/src/document_loaders/web/notionapi.ts#L121

I think it's a bit more generalizable - would it make sense to rename to `onDocumentProcessed` or something similar?

Reply:
Updated in the new commit